vignettes/articles/modify-functions.Rmd

```diff
@@ -0,0 +1,197 @@
+---
+title: "Modifier Functions"
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(gtsummary)
+theme_gtsummary_compact()
+```
+
+## Introduction
+
+Oftentimes when a table is created using {gtsummary} the default formatting does not exactly match a user's preferred formatting. To make modifications to tables after the table has been built, the `modify_*()` functions were created.
+
+In the following sections we briefly outline each of these functions that can be used to modify gtsummary tables.
+
+Many of the modifier functions share the common `columns` argument. When this argument is required, the column names provided should match their names in the table's `table_body` element (i.e. `x$table_body`). These column names can be printed to the console by running `show_header_names(x)`.
+
+When the `rows` argument is available for a modifier function, a subset of rows in the table on which the function should operate can also be specified via a predicate expression selecting rows from `x$table_body`. See the function documentation for details.
+
+We will begin by crafting a basic table to summarize a regression model. In the following sections, we will showcase how to use a variety of table modifier functions by applying them to this basic table.
+
+```{r tbl}
+x <- glm(response ~ age + grade, trial, family = binomial()) |>
+  tbl_regression(exponentiate = TRUE)
+x
+```
+
+## Column Formatting Modifiers
+
+- `modify_column_alignment()`: Update column alignment/justification
+- `modify_column_indent()`: Update column indentation
```

Comment:
Gahhh, I've been having regrets about this function name ever since this function was introduced ~6 months ago. The indentation is really on the cells, not the column.

Do you think we should change the name now? I can't image that many people have found it in these few months, and we'll go through a deprecation process.

Reply:
@ddsjoberg will update the function name and deprecate the former